src/components/LocationListView.js

```diff
@@ -87,17 +87,27 @@ export default class LocationListView extends Component {
 
     render() {
         const primary = this.props.primary;
-        if (this.props.rows.length === 0) {
+
+        if (this.props.rows.length == 0 && this.props.loaded) {
+            return (
+                <View style={styles.container}>
+                    <View style={styles.header}>
+                        <Text style={styles.headerText}>No locations found</Text>
+                    </View>
+                </View>
+            );
+        }
+        else if (!this.props.rows.length && !this.props.loaded) {
             return (
-                <View style={styles.noLocationsContainer}>
-                    <Text
-                        style={styles.noLocationsText}
-                    >
-                            There are no locations
-                    </Text>
+                <View style={styles.container}>
+                    <View style={styles.header}>
+                        <Text style={styles.headerText}>Loading locations, please wait...</Text>
```

Comment:
Same as in https://github.com/theirc/app.refugee.info/pull/93/files#diff-c937f2a56d3dfc79322d5c7f769d9b44R95